Source/Generator.cs

```diff
@@ -209,70 +222,81 @@ public static ConfigNode GenerateBody(Planet planet, String folder, String refer
             node.AddValue("randomMainMenuBody", "" + (referenceBody == "Sun"));
 
             // Template
-            ConfigNode template = new ConfigNode("Template");
+            var template = new ConfigNode("Template");
             node.AddConfigNode(template);
             if (planet != Kerbin)
             {
                 if (planet.gas_giant)
+                {
                     template.AddValue("name", "Jool");
+                }
                 else
                 {
                     template.AddValue("name", Utility.GetTemplate(planet.surface_pressure > 0.00001, false));
-                    template.AddValue("removeAllPQSMods", "True");
+                    template.AddValue("removeAllPQSmods", "True");
```

Comment:
You know that you changed a value that is later read by Kopernicus? Kopernicus is case sensitive